wcfsetup/install/files/lib/system/template/plugin/DateModifierTemplatePlugin.class.php

```diff
@@ -37,10 +36,7 @@ public function execute($tagArgs, TemplateEngine $tplObj)
         }
 
         if (!empty($tagArgs[1])) {
-            return DateUtil::format(
-                $dateTime,
-                $tagArgs[1]
-            );
+            return $dateTime->format($tagArgs[1]);
```

Comment:
I believe this one might not be correct when the timezone of the given `$dateTime` differs from the timezone of the logged-in user. To add insult to injury, `DateUtil::format()` changed the timezone for a mutable `DateTime` without cloning, so this might actually result in a behavioral change.